tlslite/extensions.py

```diff
@@ -1567,6 +1567,55 @@ def parse(self, parser):
         return self
 
 
+class ServerKeyShareExtension(TLSExtension):
+    """
+    Class for handling the Server Hello variant of the Key Share extension.
+
+    Extension for sending the key shares to client
+    """
+
+    def __init__(self):
+        """Create instance of the object."""
+        super(ServerKeyShareExtension, self).__init__(extType=ExtensionType.
+                                                      key_share,
+                                                      server=True)
+        self.server_share = None
+
+    def create(self, server_share):
+        """Set the advertised server share in the extension."""
+        self.server_share = server_share
+        return self
+
+    @property
+    def extData(self):
+        """Serialise the payload of the extension"""
+        if self.server_share is None:
+            return bytearray(0)
```

Comment:
No, that's actually a malformed key_share extension - allowed with tlsfuzzer in mind